app/utils/transactionHandlers/ConfigureDelegationHandler.ts

```diff
@@ -0,0 +1,67 @@
+import AccountTransactionDetails from '~/components/Transfers/AccountTransactionDetails';
+import ConcordiumLedgerClient from '~/features/ledger/ConcordiumLedgerClient';
+import { AccountPathInput, getAccountPath } from '~/features/ledger/Path';
+import {
+    AccountTransactionHandler,
+    TransactionExportType,
+} from '~/utils/transactionTypes';
+import {
+    AccountTransaction,
+    ConfigureDelegation,
+    instanceOfConfigureDelegation,
+    TransactionPayload,
+} from '../types';
+import { serializeTransferPayload } from '../transactionSerialization';
+
+export default class ConfigureDelegationHandler
```

Comment:
Maybe we could, however we would have to make the types not fit specifically to the transaction they handle. If anything we should make an abstract class for for all account transactions, as most of the different functions for handling transactions already implement some sort of pattern that switches based on the transaction type.

I think it's best to do this in another PR though, as this one is already big enough as is (and it would be nice to have it done soon 😄)

Reply:
please do